topo/node/cptx/cptx.go

```diff
@@ -26,18 +38,132 @@ func New(pb *topopb.Node) (node.Implementation, error) {
 
 type Node struct {
 	pb *topopb.Node
+	cliConn *scraplinetwork.Driver
 }
 
 func (n *Node) Proto() *topopb.Node {
 	return n.pb
 }
 
+// WaitCLIReady attempts to open the transport channel towards a Network OS and perform scrapligo OnOpen actions
+// for a given platform. Retries indefinitely till success.
+func (n *Node) WaitCLIReady() error {
```

Comment:
Rename it to IsTransportReady maybe ?

Reply:
Hi @shubh90, thanks for the review. It not only checks for the transport ready, but waits and retries as well; hence the name.